plot-api/src/main/kotlin/jetbrains/letsPlot/scale/DateTime.kt

```diff
@@ -37,8 +37,8 @@ fun scale_x_datetime(
     breaks: List<Any>? = null,      // ToDo: should understand Date
     labels: List<String>? = null,
     limits: Pair<Number?, Number?>? = null,
```

Comment:
For date-time scales `limits` and `naValue` can be defined in some kind of date-time units - let's have `Any` instead of `Number`.